examples/14-shadowvolumes/shadowvolumes.cpp

```diff
@@ -2357,6 +2357,10 @@ class ExampleShadowVolumes : public entry::AppI
 				bgfx::dbgTextPrintf(8, row++, 0x0f, "Stencil buffer     - Faster, but capable only of +1 incr.");
 				bgfx::dbgTextPrintf(8, row++, 0x0f, "Texture as stencil - Slower, but capable of +2 incr.");
 			}
+			else
+			{
+				bgfx::dbgTextClear();
```

Comment:
Currently if you toggle m_showHelp on/off the text would still be hanging on screen, this fixes that